pom.xml

```diff
@@ -47,46 +53,29 @@
     <tag>${scmTag}</tag>
   </scm>
 
-  <licenses>
-    <license>
-      <name>MIT License</name>
-      <url>http://opensource.org/licenses/MIT</url>
-    </license>
-  </licenses>
-
   <properties>
     <revision>1.29</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.73.3</jenkins.version>
+    <jenkins.version>2.164.1</jenkins.version>
     <java.level>8</java.level>
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>structs</artifactId>
-        <version>1.14</version>
-      </dependency>
-      <dependency>
-        <groupId>org.jenkins-ci.plugins.workflow</groupId>
-        <artifactId>workflow-step-api</artifactId>
-        <version>2.16</version>
-      </dependency>
-      <dependency>
-        <groupId>org.hamcrest</groupId>
-        <artifactId>hamcrest-core</artifactId>
-        <version>1.3</version>
+        <groupId>io.jenkins.tools.bom</groupId>
+        <artifactId>bom-2.164.x</artifactId>
+        <version>9</version>
```

Comment:
```suggestion
        <version>10</version>
```

?

Reply:
requires `2.164.3` baseline but should be fine.